src/transformer/fields/date/date.field.ts

```diff
@@ -16,13 +16,17 @@ const FRENCH_DATE_REG_EXP: RegExp = /^(?<day>\d{2})\/(?<month>\d{1,2})\/(?<year>
 
 const FRENCH_DATE_TIME_REG_EXP: RegExp = /^(?<day>\d{2})\/(?<month>\d{2})\/(?<year>\d{4}) (?<time>\d{2}:\d{2}:\d{2})$/u;
```

Comment:
```suggestion
const FRENCH_DATE_TIME_REG_EXP: RegExp = /^(?<day>\d{2})\/(?<month>\d{2})\/(?<year>\d{4}) (?<time>\d{2}:\d{2}(?::\d{2}))$/u;
```
Est-ce possible de rendre les secondes facultatives dans la regexp plutôt que dans créer une autre ?